src/SeExpr/CMakeLists.txt

```diff
@@ -94,9 +95,3 @@ ENDIF()
 ## Traverse subdirectories
 ADD_SUBDIRECTORY (src/SeExpr)
 ADD_SUBDIRECTORY (src/SeExprEditor)
-ADD_SUBDIRECTORY (src/doc)
```

Comment:
Why did you remove those?

Reply:
Just because I didn't think they were used for our case and so they didn't need to be built

Reply:
Ok. I would prefer if we keep the CMake setup as untouched as possible. I also find it useful to build the docs, demos and tests.